ffi/framebuffer_mxcfb.lua

```diff
@@ -574,7 +574,7 @@ function framebuffer:init()
 
         -- new devices
         local is_new = false
-        if self.device.model == "Cervantes2013" or self.device.model == "Cervantes3"
+        if self.device.model == "Cervantes2013" or self.device.model == "Cervantes 3"
             or self.device.model == "Cervantes4" then is_new = true end
```

Comment:
In that case there's also something off about this line, isn't there?

https://github.com/koreader/koreader/blob/4ae43aee2c084e31aa1134a2c92d20f7681fdfcc/frontend/device/cervantes/device.lua#L55

Reply:
Yeah; that's what I was wondering.  I'm happy to modify the other two as well but wasn't sure if I was missing something.